emscripten.py

```diff
@@ -972,5 +981,5 @@ def create_pointer_conversion_wrappers(metadata):
   return wrappers
 
 
-def run(in_wasm, out_wasm, outfile_js, memfile, js_syms):
-  emscript(in_wasm, out_wasm, outfile_js, memfile, js_syms)
+def run(in_wasm, out_wasm, outfile_js, memfile, js_syms, finalize=True):
+  emscript(in_wasm, out_wasm, outfile_js, memfile, js_syms, finalize)
```

Comment:
Is there any point in having this wrapper anymore?  Should we just call `emscript` from `emcc.py`?

Reply:
I'm not sure, are there users running emscripten.py by itself without emcc.py?

Reply:
There should no users other than what you see in the repo.  i.e. importing these python scripts yourself is not a supported thing.

Anyway, this is unrelated this PR so no need to figure it out here.